code/obj/item.dm

```diff
@@ -984,6 +984,12 @@
 		equipment_proxy.additive_slowdown -= spacemove
 		equipment_proxy.space_movement -= spacemove
 
+/obj/item/proc/after_unequip(var/mob/user)
```

Comment:
`unequipped()` has this quirk where the item is technically still in the slot during the proc call.
I suspect a _shitton_ of things rely on this behavior, so I added `after_unequip()`. 
See `code/mob/living/carbon/human.dm` and `code/datums/hud/human.dm` on how it's called.